src/pages/share/index.tsx

```diff
@@ -23,10 +23,10 @@ const SharePage: FC = () => {
     )
   return (
     <StyledCentered>
-      <Typography variant="h5">
+      <Typography variant="h5" style={{ border: `3px solid yellow` }}>
```

Comment:
Please remove these after test is done

Reply:
Sorry. Handled in [33e9745](https://github.com/ajktown/wordnote/pull/172/commits/33e9745905a09891f7c9a8995826f8d09e372030)